roles/addons/diskless/files/disklessset.py

```diff
@@ -132,7 +133,44 @@ def generate_ipxe_boot_file(image_type, image_name, image_kernel, image_initramf
 boot
 '''.format(image_name=image_name, image_kernel=image_kernel, image_initramfs=image_initramfs)
         return boot_file_content
-    elif image_type == 'livenet':
+    elif image_type == 'livenet' and selinux_option == 'yes':
```

Comment:
Since only thing to change is selinux=0/1, maybe it would be simpler to do a simple:

```
if selinux_option == 'yes':
    selinux = 1
else:
    selinux = 0
```

Then merge both boot_file_content ?

Reply:
Done.